GameOfLife.java

```diff
@@ -84,31 +109,68 @@ private static int[][] evolve(int[][] board) {
 	// Assumes that i is at least 1 and at most the number of rows in the board - 1. 
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
-	private static int cellValue(int[][] board, int i, int j) {
+	public static int cellValue(int[][] board, int i, int j) {
 		//// Replace the following statement with your code.
-		return 0;
+		int alive = count(board, i, j);
+		int value = board[i][j]; 
+		if (board[i][j] == 1){
+			if ((alive < 2) || (alive > 3)){
+				value = 0;
+			}
+		}
+		else if (board[i][j] == 0) {
```

Comment:
If it's not 1 than it's 0 - no need to add `if` to this `else`
```suggestion
		else {
```